spec/features/user_cannot_review_an_item_not_purchased_spec.rb

```diff
@@ -14,6 +14,6 @@
     line_items = create_list(:line_item, 1)
     order.line_items << line_items
     sign_in_as user
-    # expect(page).not_to have_link 'Write a review'
+    expect(page).not_to have_link 'Write a review'
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.